<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiLight</class>
 <widget class="QWidget" name="UiLight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vLayout_main">
    <item>
     <widget class="QPushButton" name="btn_basicSetting">
      <property name="text">
       <string>&gt;基础光源设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_basicSeeting" native="true">
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>光源颜色：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>光照强度：</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>121</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLayout_basicSetting">
        <item>
         <widget class="QPushButton" name="btn_lightColor">
          <property name="text">
           <string>选择hsv颜色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_lightIntensity">
          <property name="text">
           <string>10.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_location">
      <property name="text">
       <string>&gt;位置（点光源/面光源/聚光灯）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_location" native="true">
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>位置 x：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>        y：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>        z：</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>121</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLayout_location">
        <item>
         <widget class="QLineEdit" name="lineEdit_lightLocateX">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_lightLocateY">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_lightLocateZ">
          <property name="text">
           <string>3.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_attenuation">
      <property name="text">
       <string>&gt;衰减因子（点光源/面光源/聚光灯）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_attenuation" native="true">
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>二次项：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>一次项：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>常数项：</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>121</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLayout_attenuation">
        <item>
         <widget class="QLineEdit" name="lineEdit_quadratic">
          <property name="text">
           <string>0.032</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_liner">
          <property name="text">
           <string>0.09</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_constant">
          <property name="text">
           <string>1.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_direction">
      <property name="text">
       <string>&gt;光源方向（方向光源/面光源/聚光灯）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_direction" native="true">
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>方向 x:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>        y:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>        z:</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>121</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLayout_direction">
        <item>
         <widget class="QLineEdit" name="lineEdit_lightDirectionX">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_lightDirectionY">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_lightDirectionZ">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Angle">
      <property name="text">
       <string>&gt;内外锥角（聚光灯）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_spot" native="true">
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>内锥角：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>外锥角：</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLayout_angle">
        <item>
         <widget class="QLineEdit" name="lineEdit_lightInnerConeAngle">
          <property name="text">
           <string>45</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_lightOuterConeAngle">
          <property name="text">
           <string>60</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
